bundle/Controller/UserRegisterController.php

```diff
@@ -0,0 +1,126 @@
+use EzSystems\RepositoryForms\Form\Type\User\UserRegisterType;
+use Symfony\Component\HttpFoundation\Request;
+
+class UserRegisterController extends Controller
+{
+    /**
+     * @var ContentTypeService
+     */
+    private $contentTypeService;
+
+    /**
+     * @var UserService
+     */
+    private $userService;
+
+    /**
+     * @var ActionDispatcherInterface
+     */
+    private $contentActionDispatcher;
+
+    /**
+     * @var Repository
+     */
+    private $repository;
+
+    /**
+     * @var string
+     */
+    private $pagelayout;
+
+    public function __construct(
+        ContentTypeService $contentTypeService,
+        UserService $userService,
+        ActionDispatcherInterface $contentActionDispatcher,
+        Repository $repository
+    ) {
+        $this->contentTypeService = $contentTypeService;
+        $this->userService = $userService;
+        $this->contentActionDispatcher = $contentActionDispatcher;
+        $this->repository = $repository;
+    }
+
+    /**
+     * @param string $pagelayout
+     * @return ContentEditController
+     */
+    public function setPagelayout($pagelayout)
+    {
+        $this->pagelayout = $pagelayout;
+
+        return $this;
+    }
+
+    /**
+     * Displays and processes a user registration form.
+     *
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \Exception if the current user isn't allowed to register an account
+     */
+    public function registerAction(Request $request)
+    {
+        if (!$this->isGranted(new Attribute('user', 'register'))) {
+            throw new \Exception('You are not allowed to register a new account');
+        }
+
+        $language = 'eng-GB';
+
+        $contentType = $this->repository->sudo(
+            function () {
+                return $this->contentTypeService->loadContentTypeByIdentifier('user');
+            }
+        );
+
+        $data = (new UserCreateMapper())->mapToFormData($contentType, [
+            'mainLanguageCode' => $language,
+            'parentGroup' => $this->repository->sudo(
+                function () {
+                    return $this->userService->loadUserGroup(11);
+                }
+            ),
+        ]);
+        $form = $this->createForm(new UserRegisterType(), $data, ['languageCode' => $language]);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $this->contentActionDispatcher->dispatchFormAction($form, $data, $form->getClickedButton()->getName());
+            if ($response = $this->contentActionDispatcher->getResponse()) {
+                return $response;
+            }
+        }
+
+        return $this->render('EzSystemsRepositoryFormsBundle:Content:content_edit.html.twig', [
+            'form' => $form->createView(),
+            'languageCode' => $language,
+            'pagelayout' => $this->pagelayout,
```

Comment:
Shouldn't be a problem, the templates checks it.

Otoh, maybe it doesn't make much sense not to set it.